tasks.c

```diff
@@ -2616,7 +2631,9 @@ BaseType_t xTaskCatchUpTicks( TickType_t xTicksToCatchUp )
     /* Use xPendedTicks to mimic xTicksToCatchUp number of ticks occurring when
      * the scheduler is suspended so the ticks are executed in xTaskResumeAll(). */
     vTaskSuspendAll();
+    taskENTER_CRITICAL();

```

Comment:
vTaskSuspendAll has already shut the scheduler down so we should not need taskENTER_CRITICAL.